File: src/main/java/com/loltft/rudefriend/repository/member/MemberRepository.java
package com.loltft.rudefriend.repository.member;

import java.util.Optional;
import java.util.UUID;

import org.springframework.data.jpa.repository.JpaRepository;

import com.loltft.rudefriend.entity.Member;

public interface MemberRepository extends JpaRepository<Member, UUID>, MemberRepositoryCustom {

  Optional<Member> findByMemberId(String memberId);

  Optional<Member> findByRefreshToken(String refreshToken);
}

File: src/main/java/com/loltft/rudefriend/repository/member/MemberRepository.kt
package com.loltft.rudefriend.repository.member

import com.loltft.rudefriend.entity.Member
import org.springframework.data.jpa.repository.JpaRepository
import java.util.*

interface MemberRepository : JpaRepository<Member?, UUID?>, MemberRepositoryCustom {
    fun findByMemberId(memberId: String?): Optional<Member?>

    fun findByRefreshToken(refreshToken: String?): Optional<Member?>
}
